lib/matplotlib/figure.pyi

```diff
@@ -16,7 +16,7 @@ from matplotlib.backend_bases import (
 from matplotlib.colors import Colormap, Normalize
 from matplotlib.colorbar import Colorbar
 from matplotlib.cm import ScalarMappable
-from matplotlib.gridspec import GridSpec, SubplotSpec, SubplotParams as SubplotParams
```

Comment:
This is accidentally reverting the move of `SubplotParams`, isn't it?

Reply:
Fixed, yeah, merge conflict and I apparently resolved in the wrong direction.